src/Trixi2Vtk.jl

```diff
@@ -6,7 +6,7 @@ using HDF5: h5open, attributes, haskey
 using ProgressMeter: @showprogress, Progress, next!
 using StaticArrays: SVector
 using TimerOutputs
-using Trixi: Trixi, transfinite_mapping, coordinates2mapping, polynomial_interpolation_matrix,
+using Trixi: Trixi, transfinite_mapping, coordinates2mapping, polynomial_interpolation_matrix, (..),
```

Comment:
Could you please load it from EllipsisNotation.jl since we do not guarantee to export it from Trixi.jl?